### PR TITLE
Add `PlatformInfo`, `UpdateInfoBit` and `QuotingEnclaveReportInfo`

### PR DESCRIPTION
**Pull request by nick-mobilecoin**

### CODE REVIEW

**Comment by nick-mobilecoin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #91** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/91" target="_blank">Graphite</a> 
    * **PR #86** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/86" target="_blank">Graphite</a> 
      * **PR #92** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/92" target="_blank">Graphite</a> 
        * **PR #103** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/103" target="_blank">Graphite</a> 
          * **PR #110** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/110" target="_blank">Graphite</a> 
            * **PR #111** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/111" target="_blank">Graphite</a> 
              * **PR #112** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/112" target="_blank">Graphite</a> 
                * **PR #113** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/113" target="_blank">Graphite</a> 
                  * **PR #115** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/115" target="_blank">Graphite</a> 
                    * **PR #116** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/116" target="_blank">Graphite</a> 
                      * **PR #117** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/117" target="_blank">Graphite</a> 
                        * **PR #118** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/118" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/118?utm_source=stack-comment).

**Review by jcape — APPROVED** [review]

**Review by awygle — APPROVED** [review]

**Comment by nick-mobilecoin** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/118?utm_source=stack-comment).


Job Reference: WM00OUpxmxIPJavmjHZu

**Pull request merged**
